lib/actions/ssh.coffee

```diff
@@ -44,6 +45,11 @@ module.exports =
 			boolean: true
 			description: 'increase verbosity'
 			alias: 'v'
+	,
+			signature: 'host'
+			boolean: true
+			description: 'access host OS (for devices with Resin OS >= 2.7.5)'
+			alias: 'H'
```

Comment:
We try to share command line arguments in the CLI e.g. in https://github.com/resin-io/resin-cli/blob/master/lib/actions/command-options.coffee when they're used in multiple places - I think it'd be tidier to do that here for this and `local ssh`.

Notably, doing so highlights a difference: `local ssh` uses `-s` as the host alias, not `-H`. It'd be good to keep this consistent, and we can't change the `local ssh` format without a breaking change, so I think this should be `-s` too (though I wouldn't be surprised if we end up changing them both of together later, once we get to the big CLI redesign).